### PR TITLE
Add blog post for time travel in Delta Lake connector

### PR DESCRIPTION
**Pull request by ebyhr**

### CODE REVIEW

File: _posts/2024-04-11-time-travel-delta-lake.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+Time travel as a feature has a number of practical use cases:
+
+1. **Auditing and compliance**: Time travel enables auditors and compliance
+   teams to analyze data changes over time, ensuring regulatory compliance and
+   providing transparency into data operations.
+2. **Historical analysis**: Data analysts and data scientists can perform
+   historical analysis by querying data at different points in time, uncovering
+   trends, patterns, and anomalies that may not be apparent in current data.
+3. **Data recovery and rollback**: In the event of data corruption or erroneous
```
Comment (colebow) [inline _posts/2024-04-11-time-travel-delta-lake.md:27]: I thinks this should be #1... probably the most generally applicable.

**Review by colebow — APPROVED** [review]
Only one comment, awesome blog post.

File: _posts/2024-04-11-time-travel-delta-lake.md (hunk 1 above)
Comment (mosabua) [reply _posts/2024-04-11-time-travel-delta-lake.md:27]: yeah .. good point

**Review by mosabua — APPROVED** [review]
Excellent .. I updated a few minor things and added info about catalog, schema and such to make it consistent in terms of tables. Merging and announcing shortly.

**Pull request merged**